kano_updater/status.py

```diff
@@ -77,6 +78,7 @@ def load(self):
                 data['last_update']
                 data['last_check']
                 data['last_check_urgent']
+                data['first_boot_countdown']
```

Comment:
Given that we are adding this `first_boot_countdown` field, are we sure that this won't ever load the old version of the file and fail?